EntityFrameworkCore.Manipulation.Extensions/Internal/Extensions/SqlCommandBuilderExtensions.cs

```diff
@@ -52,44 +51,12 @@ internal static class SqlCommandBuilderExtensions
             foreach (TEntity entity in entities)
             {
                 stringBuilder.Append(" (");
-				foreach (IProperty property in properties)
-				{
-					if (useParameterPlaceholders)
-					{
-						// for each column value, create a placeholder, e.g. "{3}" used for the parameter
-						stringBuilder.Append("{").Append(parameters.Count).Append("},");
-						parameters.Add(property.PropertyInfo.GetValue(entity));
-					}
-					else
-					{
-						object value = property.PropertyInfo.GetValue(entity);
-						Type type = property.PropertyInfo.PropertyType;
-						var nullableUnderlyingType = Nullable.GetUnderlyingType(type);
-
-						var parameter = new SqlParameter($"value{parameters.Count}", value);
-
-						if (value == null)
-						{
-							parameter = new SqlParameter(parameter.ParameterName, (object)DBNull.Value);
-						}
-
-						if (nullableUnderlyingType != default)
-						{
-							type = nullableUnderlyingType;
-						}
-
-						if (type.Equals(typeof(DateTime)))
-						{
-							parameter.SqlDbType = SqlDbType.DateTime2;
-						}
-
-						// for each column value, create a paramet placeholder, e.g. "@value_5" used for the parameter
-						stringBuilder.Append("@").Append(parameter.ParameterName).Append(",");
-
-						// then add the actual value to the list of parameters
-						parameters.Add(parameter);
-					}
-				}
+                foreach (IProperty property in properties)
+                {
+                    // for each column value, create a placeholder, e.g. "{3}" used for the parameter
+                    stringBuilder.Append("{").Append(parameters.Count).Append("},");
```

Comment:
FYI, you can use `stringBuilder.AppendFormat(CultureInfo.InvariantCulture, ....)`